### PR TITLE
RDSTF-401 fix ci pipeline

### PR DESCRIPTION
**Pull request by MarioPires**
### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

File: build/ci/azure-pipeline-test.yml [hunk 1]
```diff
@@ -13,83 +13,84 @@ resources:
   repositories:
     - repository: MobilePluginsE2ETests
       type: github
-      ref: main
+      ref: RDSTF-401-fix-ci-template
```
Comment (MarioPires) [inline build/ci/azure-pipeline-test.yml]: TODO: revert back to main once this PR is merged:
https://github.com/OutSystems/MobilePluginsE2ETests/pull/8

File: build/ci/azure-pipeline-test.yml [hunk 2]
```diff
@@ -13,83 +13,84 @@ resources:
...
       name: OutSystems/MobilePluginsE2ETests
       endpoint: OutSystems
     - repository: MobilePluginsODCPipeline
       type: github
-      ref: main
+      ref: RDSTF-401-fix-mobile-package-creation
```
Comment (MarioPires) [inline build/ci/azure-pipeline-test.yml]: revert back to main once this PR is merged:
https://github.com/OutSystems/MobilePluginsODCPipeline/pull/6

File: build/ci/azure-pipeline-test.yml [hunk 3]
```diff
@@ -13,83 +13,84 @@ resources:
...
       name: OutSystems/MobilePluginsODCPipeline
       endpoint: OutSystems
 
+variables:
+  - name: StampsInformationFileName
+    value: "eng-osrd-mobile-neo1.StampsInformation.json"
+  - name: PLUGIN_APP_KEY
+    value: "7acb2c1d-51bf-4ef4-8cb8-a05573501b6a" # Camera Plugin
```
Comment (sandrarodrigues76) [inline build/ci/azure-pipeline-test.yml]: can you please add the comment ODC Camera Plugin?

File: build/ci/azure-pipeline-test.yml [hunk 4]
```diff
@@ -13,83 +13,84 @@ resources:
...
+  - name: SAMPLEAPP_APP_KEY
+    value: "15d54f75-779d-4cae-948f-1bc119a7e813" # CloneOfCameraSampleApp
```
Comment (sandrarodrigues76) [inline build/ci/azure-pipeline-test.yml]: can you please add the comment ODC CloneOfCameraSampleApp

File: build/ci/azure-pipeline-test.yml (hunk 3 above)
Comment (MarioPires) [reply build/ci/azure-pipeline-test.yml]: will do

File: build/ci/azure-pipeline-test.yml (hunk 4 above)
Comment (MarioPires) [reply build/ci/azure-pipeline-test.yml]: will do

**Review by sandrarodrigues76 — APPROVED** [review]
LGTM

**Review by sandrarodrigues76 — APPROVED** [review]
LGTM

**Review by OS-ricardomoreirasilva — APPROVED** [review]

**Pull request merged**